lib/index.d.ts

```diff
@@ -17,7 +17,7 @@ export interface SctpSettings {
 
 // Functions
 export function preload(): void;
-export function initLogger(level: LogLevel): void;
```

Comment:
I suggest changing the first parameter of the callback to a `LogLevel` instead of a number for consistency. This way, the callback uses the same levels as the `initLogger` function itself.